### ISSUE TITLE
TS declaration file broken

### ISSUE THREAD

**Issue by marvinroger** [termhn/nano-base32#1]
Hi!

The TS declaration file is broken since TypeScript 2.6 : https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#arbitrary-expressions-are-forbidden-in-export-assignments-in-ambient-contexts

```
The expression of an export assignment must be an identifier or qualified name in an ambient context.
```

Something like this should fix the issue:

```ts
declare module 'nano-base32' {
  export function encode(view: Uint8Array): string
  export function decode(input: string): Uint8Array
}
```

**Comment by fu5ha** [comment]
Ah, thanks for the info... I don't use TS myself so I sort of just made a best guess based on reference :P If you could make a PR that works I'd be happy to merge it!

**Comment by fu5ha** [comment]
Fixed in #2

**Comment by fu5ha** [comment]
Also published 1.0.1 which it should be fixed in.

**Comment by marvinroger** [comment]
Thanks Gray!
